.circleci/config.yml

```diff
@@ -216,160 +219,53 @@ commands:
-                    # The zip file has a root directory, so we put it somewhere else first before placing the files in .local
-                    unzip cmake.zip -d /tmp > /dev/null
-                    cp /tmp/cmake-<< pipeline.parameters.cmake_version >>-windows-x86_64/* -R "$HOME/.local"
-                  fi
-
-                  cmake --version
-  install_protoc:
+  # Install mise for tool version management
+  install_mise:
     parameters:
       platform:
         type: executor
     steps:
       - when:
           condition:
             or:
+              - equal: [ *amd_linux_helm_executor, << parameters.platform >> ]
               - equal: [ *amd_linux_build_executor, << parameters.platform >> ]
               - equal: [ *amd_linux_test_executor, << parameters.platform >> ]
-          steps:
-            - run:
-                name: Install protoc
-                command: |
-                  if [[ ! -f "$HOME/.local/bin/protoc" ]]; then
-                    curl -L https://github.com/protocolbuffers/protobuf/releases/download/v<< pipeline.parameters.protoc_version >>/protoc-<< pipeline.parameters.protoc_version >>-linux-x86_64.zip --output protoc.zip
-                    unzip protoc.zip -d $HOME/.local
-                  fi
-      - when:
-          condition:
-            or:
               - equal: [ *arm_linux_build_executor, << parameters.platform >> ]
               - equal: [ *arm_linux_test_executor, << parameters.platform >> ]
-          steps:
-            - run:
-                name: Install protoc
-                command: |
-                  if [[ ! -f "$HOME/.local/bin/protoc" ]]; then
-                    curl -L https://github.com/protocolbuffers/protobuf/releases/download/v<< pipeline.parameters.protoc_version >>/protoc-<< pipeline.parameters.protoc_version >>-linux-aarch_64.zip --output protoc.zip
-                    unzip protoc.zip -d $HOME/.local
-                  fi
-      - when:
-          condition:
-            or:
               - equal: [ *macos_build_executor, << parameters.platform >> ]
               - equal: [ *macos_test_executor, << parameters.platform >> ]
           steps:
             - run:
-                name: Install protoc
+                name: Install mise
                 command: |
-                  if [[ ! -f "$HOME/.local/bin/protoc" ]]; then
-                    curl -L https://github.com/protocolbuffers/protobuf/releases/download/v<< pipeline.parameters.protoc_version >>/protoc-<< pipeline.parameters.protoc_version >>-osx-universal_binary.zip --output protoc.zip
-                    unzip protoc.zip -d $HOME/.local
-                  fi
+                  curl https://mise.jdx.dev/install.sh | sh
+                  mise activate bash >> "$BASH_ENV"
+                  mise trust
```

Comment:
This shouldn't be necessary in CI

Reply:
interesting.  didn't know that, but makes sense.  will remove it in another PR.